CODEOWNERS

```diff
@@ -0,0 +1,17 @@
+# TensorFlow Addons Codeowners
+
+# SIG maintainers are the catch all for reviews
+* @tensorflow/sig-addons-maintainers
+
+
+# Subpackage Owners
+seq2seq/* @qlzh727
```

Comment:
`/tensorflow_addons/seq2seq`?

Reply:
So seq2seq is also located in custom_op which is why I used this syntax. https://help.github.com/en/articles/about-code-owners#example-of-a-codeowners-file

Reply:
This synatx will not match further nested files, say `custom/seq2seq/cc/kernel`. Perhaps we can list them line by line: 

```
/tensorflow_addons/seq2sqe qlzh727
/tensorflow_addons/custom_ops/seq2seq qlzh727
```

Reply:
Ah you're correct I mis-read that example. I'll submit a change in a bit here... I think verbosely listing out all managed directories is fine.